music-quiz/src/presenters/gamePresenter.js

```diff
@@ -181,7 +182,7 @@ function GamePresenter(props) {
           <GameChallengeView
             playlist={challengePlaylistPromiseState.data}
             onStartClick={startGame}
-            onDeclineClick={declineChallenge}
+            onDeclineClick={clearGameSettings}
```

Comment:
Do we not need to remove the request if it is declined? Like we did in "declineChallenge" (removed now).

Reply:
Tested if I was able to decline a challenge and checked that the pending request was removed. This works so I guess this is not an issue and that you solve this another way 👍